server.js

```diff
@@ -6,7 +6,7 @@ const StatusCodes = require("http-status-codes").StatusCodes;
 
 
 const app = express();
-const port = parseInt(process.env.PORT, 10) || 8080;
+const port = parseInt(process.env.PORT, 10) || 8080 || 3000;
```

Comment:
Assuming the left-most parameter is null, 8080 will always be chosen over 3000. 

Do you want to use 3000 instead of 8080?